support-frontend/assets/pages/weekly-subscription-landing/components/content/benefits.tsx

```diff
@@ -38,7 +30,7 @@ function Benefits({
 					content: (
 						<>
 							<BenefitsHeading text="As a subscriber you’ll enjoy" />
-							<List items={getBenefits(countryGroupId)} />
+							<List items={getBenefits()} />
```

Comment:
Looks like we don't need a `getBenefits` function / function call anymore, we could just pass down the `coreBenefits` list in the `items` prop here.

Reply:
ah yes, done.